sourcetracker/_sourcetracker.py

```diff
@@ -379,11 +379,11 @@ def __init__(self, alpha1, alpha2, beta, source_data):
         self.joint_probability = np.zeros(self.V, dtype=np.float64)
 
     def set_n(self, n):
-        """Set the sum of the sink."""
+        '''Set the sum of the sink.'''
```

Comment:
A setter like this isn't necessary, since `n` is a public property and can be set directly. I recommend dropping the setter, but not a big deal.